tests/integration/scripts/deploy-latest-release-to-cluster.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+RELEASE_INFO=$(curl -L \
+                 -H "Accept: application/vnd.github+json" \
+                 -H "X-GitHub-Api-Version: 2022-11-28" \
+                 https://api.github.com/repos/kyma-project/istio/releases/latest)
+
+RELEASE_MANIFEST_URL=$(echo "$RELEASE_INFO" | jq -r '.assets[] | select(.name == "istio-manager.yaml") | .browser_download_url')
+curl -L "$RELEASE_MANIFEST_URL" | kubectl apply -f -
```

Comment:
I think that using `git describe --tags --abbrev=0` command to get the latest tag would be a better option, as the command respects branch history (e.g. we will get last patch version on release branch). Then we could just `curl -L https://github.com/kyma-project/istio/releases/download/${TAG}/istio-default-cr.yaml`